R/gamma.R

```diff
@@ -0,0 +1,58 @@
+#' gamma
+#'
+#' @param x numeric vector of quantiles
+#' @param shape shape parameter of gamma distribution
+#' @param rate rate parameter of gamma distribution, default 1
+#' @param shift shift of the left end of the distribution
+#' @param ... additional parameters to be passed to stats::dgamma
+#'
+#' @return density of distribution
+#' @export
+dgamma <- function(x, shape, rate = 1, shift = 0, ...) {
```

Comment:
I don't know if I should be commenting about the gamma stuff here but I would do the same `@rdname gamma` to link all the pages together.

Reply:
I added this to the gamma branch :)